src/controllers/events.js

```diff
@@ -0,0 +1,39 @@
+const Event = require('../models/event')
+const { uploadImage } = require('../db/storage')
+const { getFileExtension } = require('../utility/utils');
+const {EVENT_IMAGE_DIR} = require('../utility/variables')
+
+const createEvent = async (req, res) => {
+    try {
+        const imageUrl = await uploadImage(
+            req.file,
+            `${EVENT_IMAGE_DIR}/${Date.now()}.${getFileExtension(
```

Comment:
What is Date.now() my friend? 
The name of the file should be the id of the event so we know to which event an image belongs. 
In the beginning, you don't have the id, so you have to postpone uploading the image until after you create the event.